takeshape/penny
# Do not allow selection of oos variants

### Test Plan (Steps to test):

1. Go to An Exceptional T-Shirt
1. Note that you cannot add any out of stock variants — Black / XS is not allowed.
1. Try to somehow get the OOS variant in your cart...
2. Find the other OOS variant and report back
